src/pdf/ShoppingList/ShoppingListButton.tsx

```diff
@@ -0,0 +1,24 @@
+"use client";
```

Comment:
This will probably be combined to the generic pdf component.

Reply:
another race to see which gets merged first :)